projects/postgresql/build.sh

```diff
@@ -14,12 +14,24 @@
 # limitations under the License.
 #
 ################################################################################
-git apply ../add_fuzzers.diff
 cp -r $SRC/fuzzer src/backend/
-mkdir bld
+
+useradd fuzzuser
```

Comment:
Isn't it ok to run this under the current user?

Reply:
Postgresql requires a non-root user to create a database